File: Manifest_Management_Backend/Entities/Models/Crewoperator.cs
using System;
using System.Collections.Generic;

namespace ManifestPortal1.Entities.Models;

public partial class Crewoperator
{
    public int CrewId { get; set; }

    public string? Salutation { get; set; }

    public string? FirstName { get; set; }

    public string? LastName { get; set; }

    public string? ContactNumber { get; set; }

    public DateTime? Dob { get; set; }

    public string? Gender { get; set; }

    public string? CrewUsername { get; set; }

    public string? Email { get; set; }

    public string? Password { get; set; }

    public byte[]? CrewImage { get; set; }
}

File: Manifest Management_Andriod/app/src/main/java/com/sko/manifestmanagement/model/CrewOperator.kt
package com.sko.manifestmanagement.model

data class Crewoperator(
    val crewId: Int,
    val salutation: String,
    val firstName: String,
    val lastName: String,
    val contactNumber: String,
    val dob: String,  // You may want to use a Date type instead of String
    val gender: String,
    val crewUsername: String,
    val email: String,
    val password: String,  // This should probably not be returned in a real API response
    val crewImage: String?  // If the image URL is null, it will be handled correctly
)

